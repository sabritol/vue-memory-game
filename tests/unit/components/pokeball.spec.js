import { shallowMount } from '@vue/test-utils'
import Pokeball from '@/components/Pokeball.vue'

describe('Pokeball.vue', () => {
  it('renders component', () => {
    const wrapper = shallowMount(Pokeball)
    expect(wrapper.element).toMatchSnapshot()
  })
})
