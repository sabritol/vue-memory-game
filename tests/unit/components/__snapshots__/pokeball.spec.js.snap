// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pokeball.vue renders component 1`] = `
<div
  class="pokeball__parent"
>
  <div
    class="pokeball"
  >
    <div
      class="pokeball__main"
    >
      <div
        class="pokeball__button"
      />
    </div>
  </div>
</div>
`;
